src/legacy/legacy-default-xpring-client.ts

```diff
@@ -234,6 +234,21 @@ class LegacyDefaultXpringClient implements XpringClientDecorator {
       return feeAmount
     })
   }
+
+  public async accountExists(address: string): Promise<boolean> {
+    // TODO: replace with refactored checkNotXAddress() when complete
+    const classicAddress = Utils.decodeXAddress(address)
+    if (!classicAddress) {
+      throw new Error(LegacyXpringClientErrorMessages.xAddressRequired)
+    }
+    try {
+      await this.getBalance(address)
+      return true
+    } catch (e) {
+      await this.getBalance(address)
```

Comment:
This is a typo right? I think this will just throw another exception.

Reply:
oh yeah this totally is.  I removed this from default-xpring-client but forgot it here.  I think this was part of a linter weirdness experiment.